### PR TITLE
*/*: Remove postgres9_5

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

**Review by titanofold — APPROVED** [review]

File: dev-db/pgpool2/pgpool2-9999.ebuild [hunk 1]
```diff
@@ -5,18 +5,16 @@ EAPI=7
...
+POSTGRES_COMPAT=( 9.6 {10..13} )
 
 inherit autotools git-r3 postgres-multi
 
 DESCRIPTION="Connection pool server for PostgreSQL"
 HOMEPAGE="https://www.pgpool.net/"
-SRC_URI=""
+
 LICENSE="BSD"
 SLOT="0"
```
Comment (SoapGentoo) [inline dev-db/pgpool2/pgpool2-9999.ebuild:17]: there's still an empty line here?

File: dev-db/pgpool2/pgpool2-9999.ebuild [hunk 2]
```diff
@@ -25,18 +23,16 @@ RDEPEND="
 	virtual/libcrypt:=
 	memcached? ( dev-libs/libmemcached )
 	pam? ( sys-auth/pambase )
-	ssl? (
-		dev-libs/openssl:0=
-	)
+	ssl? ( dev-libs/openssl:0= )
```
Comment (SoapGentoo) [inline dev-db/pgpool2/pgpool2-9999.ebuild:26]:
```suggestion
	ssl? ( dev-libs/openssl:= )
```
the slots are pretty pointless these days

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-db/pgpool2/pgpool2-9999.ebuild (hunk 1 above)
Comment (jsmolic) [reply dev-db/pgpool2/pgpool2-9999.ebuild]: oops :D

File: dev-db/pgpool2/pgpool2-9999.ebuild (hunk 2 above)
Comment (jsmolic) [reply dev-db/pgpool2/pgpool2-9999.ebuild]: Done

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**